main.py

```diff
@@ -183,7 +190,25 @@ def run():
 
     log("Server started!")
 
+def startMonitorsIfNeeded():
+    dir = os.path.dirname(__file__)
+    prog = os.path.join(dir, 'main.py')
+    scheduler = CronScheduler()
+
+    # Temp if on RasPi
+    if c.Temp.exists():
+        log('Start monitor of CPU temp...')
+        critical_events_command = 'python3 {} --critical-events'.format(prog)
```

Comment:
This will cause issues if left as `python3` on our current system setup we have the default python set as python 3.9.9 but the python3 version is 3.8 something. Which, now that I'm thinking about it, that also needs added to the `readme.md`.

Reply:
Yeah, its because the changes have been added via v1.0.1 but v1.1 hasn't be rebased against 1.0.0 which has the changes. I need to do that. Thanks for pointing that out though.